### DISCUSSION TITLE
Has the enlightn pro repository been removed?

### DISCUSSION THREAD

**Discussion by Yuutakasan** [enlightn/enlightn#135] [Q&A]
I ran the compile today and it failed to import the enlightn pro repository and gave me an error, is there a problem occurring?

**Comment by paras-malhotra** [comment]
@Yuutakasan it's working on my side. Can you try again and ensure that you're using a key with a Pro license? If you face any issues, feel free to email support [at] laravel-enlightn [dot com]

**Reply by Yuutakasan** [reply]
Sorry, I'm not sure if it's a timing issue or not, but it's working now. It may be a timing issue, but it is working fine now.
Sorry for the strange report.
